app/src/main/res/values-hr/strings.xml

```diff
@@ -0,0 +1,628 @@
+<!--https://newbedev.com/concatenate-multiple-strings-in-xml-->
+<resources>
+    <!-- KEYS DON'T TRANSLATE -->
+    <string name="search_providers_list_key" translatable="false">search_providers_list</string>
```

Comment:
please remove all of the entries which have `translatable="false"` as they shouldn't be translated